datastream-utils/src/main/java/com/linkedin/datastream/common/zk/ZkClient.java

```diff
@@ -347,9 +216,12 @@ public void ensurePath(String path) {
     return (T) _zkSerializer.deserialize(data);
   }
 
+  /**
+   * Get Zk sessions session ID
+   */
   @VisibleForTesting
   public long getSessionId() {
```

Comment:
Is this method required anymore? Can we directly use the super method?

Reply:
Removed this method